modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -96,6 +96,11 @@ web.ssh_salt_pre_flight_script = /usr/share/susemanager/salt-ssh/preflight.sh
 
 web.ssh_use_salt_thin = false
 ----
+Create the extra drop-in configuration file for Salt Master [path]``/etc/salt/master.d/ssh-preflight.conf``:
```

Comment:
I meant in addition.  But feel free to ignore it. I'm not sure whether will cause more trouble than good:
```suggestion
Create the extra drop-in configuration file for Salt Master [path]``/etc/salt/master.d/ssh-preflight.conf`` with [literal]``ssh_run_pre_flight`` set to [literal]``true``:
```

Reply:
Oh, this way. Looks good to me :+1: